samples/hssi/hssi_100g_cmd.h

```diff
@@ -228,6 +496,49 @@ class hssi_100g_cmd : public hssi_cmd
         enable_eth_loopback(eth_ifc, false);
     }
 
+    /* Monitor Implementation */
+
+    if (monitor_ != "none")
+    {
+      uint32_t timer = 0;
+      uint32_t max_timer = stoi(monitor_);
+      char key;
+
+      uint8_t port_size = (uint8_t)port_.size();
+      sort(port_.begin(), port_.end());
+      std::vector<perf_data*> old_perf_data(port_size);    // old performance
+
+      /* Initialise performance data and initiate monitor */
+      print_monitor_headers(std::cout);
+      for(uint8_t port=0; port < port_size; port++)
+      {
+        old_perf_data[port] = (perf_data*)malloc(sizeof(perf_data));
+        select_port(port_[port], config_data, hafu);
+        capture_perf(old_perf_data[port], hafu);           // captures the monitor data for first time
+      }
+
+      /* Calculate performance and run monitor */
+      while(timer < max_timer)
```

Comment:
Handle CTL+C , CTL+Z

Reply:
add  Column name  "Time" to pref counters table  to prints seconds completed

Reply:
sure we will update it